scrapyrt/utils.py

```diff
@@ -19,3 +23,23 @@ def extract_scrapy_request_args(dictionary, raise_error=False):
                 msg = u"{!r} is not a valid argument for scrapy.Request.__init__"
                 raise ValueError(msg.format(key))
     return result
+
+
+def is_python2():
```

Comment:
Since you have six as dependency, maybe you can use six.PY2 instead.
https://pythonhosted.org/six/#six.PY2